WebService/v1/Models/SimulationApiModel/SimulationApiModel.cs

```diff
@@ -38,13 +44,22 @@ public class SimulationApiModel
         [JsonProperty(PropertyName = "EndTime", NullValueHandling = NullValueHandling.Ignore)]
         public string EndTime { get; set; }
 
+        [JsonProperty(PropertyName = "StopTime", NullValueHandling = NullValueHandling.Ignore)]
+        public string StopTime { get; set; }
+
         [JsonProperty(PropertyName = "DeviceModels")]
         public IList<SimulationDeviceModelRef> DeviceModels { get; set; }
 
+        [JsonProperty(PropertyName = "TotalMsgs", NullValueHandling = NullValueHandling.Ignore)]
+        public int TotalMessagesSent { get; set; }
+
+        [JsonProperty(PropertyName = "AvgMsgs", NullValueHandling = NullValueHandling.Ignore)]
+        public double AverageMessagesSent { get; set; }
+
         [JsonProperty(PropertyName = "$metadata", Order = 1000)]
         public IDictionary<string, string> Metadata => new Dictionary<string, string>
         {
-            { "$type", "Simulation;" + Version.NUMBER },
+            { "$type", "Simulation" },
```

Comment:
we should leave the version number, this is the model version that the UI can use to understand the schema of the response - it's the same approach across all our APIs